myia/operations/prim_random_initialize.py

```diff
@@ -0,0 +1,35 @@
+"""Definitions for the primitive `random_initialize`."""
+
+import numpy as np
+
+from ..lib import AbstractRandomState, standard_prim
+from . import primitives as P
+
+
+def pyimpl_random_initialize(seed):
+    """Implement `random_initialize`."""
+    return np.random.RandomState(seed)
+
+
+@standard_prim(P.random_initialize)
+async def infer_random_initialize(self, engine, seed):
```

Comment:
Do we require the seed to be an integer? If so, add a type check.

Reply:
Yes, seed is the key, and it should be a uint32. So I will add a typecheck.